index.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var fs = require('fs');
+var through2 = require('through2');
+var gettextParser = require('gettext-parser');
+var Handlebars = require('handlebars');
+
+
+var replaceText = function(catalog, opts, chunk, enc, callback) {
+  var template = _.template(chunk.toString(), {
```

Comment:
Why the switch from handlebars to lodash templates?

I'm not familiar with the difference between the two but if the feature set is not identical then this seems like a source for potential issue.

I've learned the hard way that sending whole files to lodash template is terribly inefficient—if the file has any size, it will exceed node's default stack size (and then you have to have nasty helper scripts to bump node's stack size).  I fully regret using it and advise not doing so here.